yb-voyager/cmd/importData.go

```diff
@@ -1103,7 +1099,7 @@ func executeSqlStmtWithRetries(conn **pgx.Conn, sqlInfo sqlInfo, objType string)
 		}
 		_, err = (*conn).Exec(context.Background(), sqlInfo.formattedStmt)
 		if err == nil {
-			utils.PrintSqlStmtIfDDL(sqlInfo.stmt)
+			utils.PrintSqlStmtIfDDL(sqlInfo.stmt, utils.GetObjectFileName(filepath.Join(exportDir, "schema"), objType))
```

Comment:
Can you just pass the `objType` to the `PrintSqlStmtIfDDL()` and find the file-name associated with it inside the `PrintSqlStmtIfDDL()`?

Reply:
@amit-yb  this function has exactly same two arguments used - sqlStmt and objectFileName